### PR TITLE
Adds two holidays.

### PR DESCRIPTION
**Pull request by OhEmGee**
Adds two holidays to holidays.dm: Remembrance day (18.5) and Civil Servant's day (20.6).
Corrects a typo.

Made in the webclient because I'm a fucking madman.

### CODE REVIEW

**Comment by Belsima** [comment]
Mother of commits.

File: code/game/gamemodes/events/holidays/Holidays.dm [hunk 1]
```diff
@@ -59,7 +59,7 @@ var/global/list/Holiday = list() //Holidays are lists now, so we can have more t
 				if(14)
 					Holiday["Pi Day"] = "An unoffical holiday celebrating the mathematical constant Pi.  It is celebrated on \
```
Comment (Atermonera) [inline code/game/gamemodes/events/holidays/Holidays.dm]: An unofficial holiday

File: code/game/gamemodes/events/holidays/Holidays.dm [hunk 2]
```diff
@@ -93,6 +93,14 @@ var/global/list/Holiday = list() //Holidays are lists now, so we can have more t
 			switch(DD)
 				if(1)
 					Holiday["Interstellar Workers' Day"] = "This holiday celebrates the work of laborers and the working class."
+				
+				if(18)
+					Holiday["Remembrance Day"] = "Rembrance Day (or, as it is more informally known, Armistice Day) is confederate-wide holiday mostly observed \
```
Comment (Atermonera) [inline code/game/gamemodes/events/holidays/Holidays.dm]: Remembrance Day [...] is a confederation-wide holiday mostly observed in its member states [...] Officially [...] Unofficially [...] Human-Unathi War [...] and the wearing of poppies

File: code/game/gamemodes/events/holidays/Holidays.dm [hunk 3]
```diff
@@ -102,9 +110,14 @@ var/global/list/Holiday = list() //Holidays are lists now, so we can have more t
 				if(6)
 					Holiday["Sapient Rights Day"] = "This holiday celebrates the passing of the Declaration of Sapient Rights by SolGov, which guarantees the \
 					same protections humans are granted to all sapient, living species."
+					
 				if(14)
 					Holiday["Blood Donor Day"] = "This holiday was created to raise awareness of the need for safe blood and blood products, \
 					and to thank blood donors for their voluntary, life-saving gifts of blood."
+				
+				if(20)
+					Holiday["Civil Servant's Day"] = "Civil Servant's day is a holiday observed in SCG member states that honors civil servants everywhere,\
```
Comment (Atermonera) [inline code/game/gamemodes/events/holidays/Holidays.dm]: Civil Servant's Day [...] or have been civil servants in the past.

File: code/game/gamemodes/events/holidays/Holidays.dm [hunk 4]
```diff
@@ -93,6 +93,14 @@ var/global/list/Holiday = list() //Holidays are lists now, so we can have more t
 			switch(DD)
 				if(1)
 					Holiday["Interstellar Workers' Day"] = "This holiday celebrates the work of laborers and the working class."
+				
+				if(18)
+					Holiday["Remembrance Day"] = "Rembrance Day (or, as it is more informally known, Armistice Day) is confederate-wide holiday mostly observed \ 
+					in it's member states since the late 2520's. Offcially, it is a day of remembering the men and women who died in various armed conflicts \
```
Comment (Anewbe) [inline code/game/gamemodes/events/holidays/Holidays.dm]: Switch this _in_ to a _by_ and it might clear up the Travis issue.

**Comment by Anewbe** [comment]
@OhEmGee Are you going to continue working on this?

File: code/game/gamemodes/events/holidays/Holidays.dm (hunk 2 above)
Comment (OhEmGee) [inline code/game/gamemodes/events/holidays/Holidays.dm]: Huh, so it WAS wearing. I wrote Wearing at first, but then corrected it.

**Comment by OhEmGee** [comment]
@Anewbe whelp, i'm fucking retarded and can't figure the conflicts out, might as well close this

**Comment by LorenLuke** [comment]
@OhEmGee 
Give me a bit... I'll have a go at adding them.

**Comment by Anewbe** [comment]
Merged the other one.

**Pull request closed without merging**